jgiven-core/src/test/java/com/tngtech/jgiven/impl/params/DefaultAsProviderTest.java

```diff
@@ -17,7 +17,8 @@ public class DefaultAsProviderTest {
             "FooBar, foo bar",
             "Foo_Bar, Foo Bar",
             "foo_bar_Baz, foo bar Baz",
-            "foo_bar, foo bar"
+            "foo_bar, foo bar",
+            "foo bar-lkjw24dsgf", "foo bar"
```

Comment:
@TimothyEarley  This line breaks the data provider, because, while it actually expects one String, not two. what you wrote counts as two separate cases.

(Understandable error though)

Reply:
Fixed!